packages/forms/src/UIForm/fields/Code/displayMode/TextMode.component.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import TextMode from './TextMode.component';
+
+describe('Code text display mode', () => {
+	const schema = {
+		title: 'My input title',
+	};
+
+	it('should render textarea', () => {
+		// when
+		const { baseElement } = render(<TextMode id="myForm" schema={schema} value="toto" />);
+
+		// then
+		expect(baseElement).toMatchSnapshot();
```

Comment:
this is just a snapshot, and I don't see any textarea in it.

Reply:
done, I missed to update test label.
I've removed snapshots